packages/components/demo/example/example.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from "@angular/core";
+import { TemplateBaseComponent } from "../../../../src/app/shared/components/template/components/base";
+import {
+  getBooleanParamFromTemplateRow,
+  getStringParamFromTemplateRow,
+} from "src/app/shared/utils";
+
+interface IExampleComponentParams {
+  /** TEMPLATE_PARAMETER: "variant". Default "red_square" */
+  variant: "red_square" | "yellow_circle";
+  /** TEMPLATE_PARAMETER: "my_boolean_param". A paramater that takes a boolean value. Default true */
+  myBooleanParam: boolean;
+  /** TEMPLATE_PARAMETER: "my_string_param". A string to be displayed */
+  myStringParam: string;
+}
+
+@Component({
+  selector: "plh-example",
```

Comment:
nit(blocking): it would be good to update this component to use the prefix `demo`, and possibly rename the component from `example` -> `basic`, so the full prefix would be `demo-basic`